crates/pipeline_manager/src/api/config_api.rs

```diff
@@ -34,5 +34,23 @@ async fn get_authentication_config(
     Ok(HttpResponse::Ok().json(&auth_config.provider))
 }
 
+/// Get the list of canned demo URLs.
+#[utoipa::path(
+    path="/config/demos",
+    responses(
+        (status = OK
+            , description = "URLs to JSON objects that self-describe a set of canned demos"
+            , content_type = "application/json"
+            , body = Vec<String>),
+    ),
+    context_path = "/v0",
+    security(("JSON web token (JWT) or API key" = [])),
+    tag = "Configuration",
+)]
+#[get("/config/demos")]
```

Comment:
Does this have to be under `/config`? Seems like demos should be a separate tag.

Reply:
The reasoning is I think that it is part of the configuration with which you start the pipeline-manager (it is provided as a command-line argument), as such you fetch it from the config.

Reply:
Yeah, I get it. I do think it's a bit of a stretch, but I'm ok with it.